### PR TITLE
Make stats.{Int,Float}64 never return errors

### PR DESCRIPTION
**Pull request by ramonza**
I left the error in the signature for now since it's used externally. We should remove it in the next round of deprecation removals.

Updates: #544

### CODE REVIEW

File: stats/measure_float64.go [hunk 1]
```diff
@@ -37,21 +29,15 @@ func (f *Float64Measure) M(v float64) Measurement {
 	return Measurement{m: f, v: v}
 }
 
-// Float64 creates a new measure of type Float64Measure. It returns
-// an error if a measure with the same name already exists.
+// Float64 creates a new measure of type Float64Measure.
+// It never returns an error.
+//TODO(ramonza): remove error
```
Comment (rakyll) [inline stats/measure_float64.go]: Move this in the block.

File: stats/measure_int64.go [hunk 2]
```diff
@@ -17,15 +17,7 @@ package stats
...
-	measure
-}
-
-func (i *Int64Measure) subscribe() {
-	i.measure.subscribe()
-}
-
-func (i *Int64Measure) subscribed() bool {
-	return i.measure.subscribed()
+	Measure
```
Comment (rakyll) [inline stats/measure_int64.go]: Why are we changing this?

File: stats/measure_int64.go [hunk 3]
```diff
@@ -37,21 +29,15 @@ func (i *Int64Measure) M(v int64) Measurement {
 	return Measurement{m: i, v: float64(v)}
 }
 
-// Int64 creates a new measure of type Int64Measure. It returns an
-// error if a measure with the same name already exists.
+// Int64 creates a new measure of type Int64Measure.
+// It never returns an error.
+//TODO(ramonza): remove error
```
Comment (rakyll) [inline stats/measure_int64.go]: Remove this in the block.

**Comment by ramonza** [comment]
@rakyll removed the embedding, PTAL

**Pull request merged**